checkout-ui-custom/src/partials/Deliver/utils.js

```diff
@@ -114,6 +114,13 @@ const populateAddressFromSearch = (address) => {
 
   // Update previously invalid fields.
   $(':invalid').trigger('change');
+  // Select the element
+  const noteField = document.querySelector('.bash--note-field');
+
+  // Hide the element after the transition
+  setTimeout(() => {
```

Comment:
Similar to the other PR, please rather use class names with CSS, instead of settimeout